.travis.yml

```diff
@@ -9,15 +9,15 @@ install:
   - go mod download
 script:
   - go test ./... -v
+  - go env
   - GOOS=linux GOARCH=amd64 go build -ldflags="-X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildVersion=$VERSION' -X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildTime=$time' -X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildCommit=$TRAVIS_COMMIT'" -o nancy-linux.amd64-$VERSION
   - GOOS=darwin GOARCH=amd64 go build -ldflags="-X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildVersion=$VERSION' -X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildTime=$time' -X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildCommit=$TRAVIS_COMMIT'" -o nancy-darwin.amd64-$VERSION
   - GOOS=windows GOARCH=amd64 go build -ldflags="-X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildVersion=$VERSION' -X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildTime=$time' -X 'github.com/sonatype-nexus-community/nancy/buildversion.BuildCommit=$TRAVIS_COMMIT'" -o nancy-windows.amd64-$VERSION.exe
   - ./nancy-linux.amd64-$VERSION Gopkg.lock
   - ./nancy-linux.amd64-$VERSION go.sum
   - go list -m all | ./nancy-linux.amd64-$VERSION
 env:
-  - GO111MODULE=on
-  - CGO_ENABLED=0
+  - GO111MODULE=on CGO_ENABLED=0
```

Comment:
Gah! Nice fix!

Prompted me to lookup this: https://docs.travis-ci.com/user/environment-variables/#defining-multiple-variables-per-item